CyclePhillyAndroid/src/main/java/org/phillyopen/mytracks/cyclephilly/tab/SlidingTabLayout.java

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright (C) 2013 The Android Open Source Project
```

Comment:
@flibbertigibbet I have retained the license for all the files that come from AOSP. Making a comment here to show that. Other files that are from AOSP are SlidingTabStrip, SlidingTabsColorsFragment and activity_main.xml